ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -47,8 +47,19 @@ logging synchronous level {{ logging.synchronous.level | arista.avd.default("cri
 {{ logging_host_cli }}
 {%         endfor %}
 {%     endfor %}
-{%     if logging.format.timestamp is arista.avd.defined('high-resolution') %}
-logging format timestamp high-resolution
+{%     if logging.format.timestamp is arista.avd.defined %}
+{%         if logging.format.timestamp is arista.avd.defined('high-resolution') %}
+{%             set timestamp_cli = "logging format timestamp high-resolution" %}
+{%         elif "traditional" in logging.format.timestamp %}
+{%             set timestamp_cli = "logging format timestamp traditional" %}
+{%             if "year" in logging.format.timestamp %}
+{%                 set timestamp_cli = timestamp_cli ~ " year" %}
+{%             endif %}
+{%             if "timezone" in logging.format.timestamp %}
+{%                 set timestamp_cli = timestamp_cli ~ " timezone"%}
```

Comment:
space required before %}

Reply:
added, now stuck at some of the mkdocs.. can you rerun it maybe?